lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -59,7 +59,6 @@ describe('i18n', () => {
     it('returns icu messages in the specified locale', () => {
       const strings = i18n.getRendererFormattedStrings('en-XA');
       expect(strings.passedAuditsGroupTitle).toEqual('[Þåššéð åûðîţš one two]');
-      expect(strings.scorescaleLabel).toEqual('[Šçöŕé šçåļé: one two]');
```

Comment:
can you replace this rather than removing it

Reply:
Done.